pkg/cmd/pulumi/stack_init.go

```diff
@@ -29,11 +30,25 @@ const (
 		"(possible choices: default, passphrase, awskms, azurekeyvault, gcpkms, hashivault)"
 )
 
+// These are the options that can be supplied during stack creation.
+type stackCreateOptions struct {
+	// Service Only: this is the list of teams what will be given permissions
+	// on the stack after creation.
+	teams []string
+}
+
+// GetTeams returns a list of teams intened to be assigned to this stack after creation.
+func (opts *stackCreateOptions) GetTeams() []string {
```

Comment:
naming nit:
Go idioms prefer not to have the "Get" prefix in the name of getters unless necessary.

```suggestion
// Teams returns a list of teams intened to be assigned to this stack after creation.
func (opts *stackCreateOptions) Teams() []string {
```

ref: https://go.dev/doc/effective_go#Getters

Reply:
Yup, though I think Pulumi doesn't adhere to this very closely. Doesn't mean we can't start now ;D SGTM!

Reply:
Implemented.